unlock-app/src/middlewares/providerMiddleware.ts

```diff
@@ -47,13 +47,9 @@ export async function initializeUnlockProvider(
   unlockProvider: any,
   dispatch: any
 ) {
-  const { key, emailAddress, password } = action
   try {
-    await unlockProvider.connect({ key, password })
+    await unlockProvider.connect(action)
```

Comment:
Could you remind me of `action` is for? Maybe put it as a comment on JSdoc?

Reply:
Yup, I'll add in that it's the `GOT_ENCRYPTED_PRIVATE_KEY` action. (there's actually just such a comment on the other side, in the provider, but it makes sense to also have it here.)